simple-cicd-configuration.yml

```diff
@@ -10,6 +10,6 @@ deploy:
     provider: script
     script: echo deploying my app 
 
-after_script:
-  - if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then git clone https://github.com/searchmetrics/dd-deploy-metric.git ; bash dd-deploy-metric/dd-deploy-metric.sh; fi
+after_deploy:
```

Comment:
Shouldn't we deploy only  after the story is delivered to production ?

Reply:
It shall be executed only when it's deployed to production.
In that case maybe it makes sense to clarify that the "deploy" phase shall be used only for production deployment.

Reply:
@amalioadam only when its production
@sm-tomislav-civcija its should be after script and not after deploy